### PR TITLE
feat: implement homepage design according to figma

### LINKED ISSUE #421: Implement homepage design according to figma
The basic files and structure is in the project, now the need is to design it according to figma design.
In order to see the page on localhost, change the end of the path to 'home'.

Link to Figma design (still needs improvement but this is the basic) :
https://www.figma.com/file/0D6JxNw691wIstb0wuR5NW/DataBus-Design?type=design&mode=design&t=TtyNCSU8UuBthoOY-0

### PR DESCRIPTION
**Pull request by itsoriki**
Resolves #421 

# Description
Implemented homepage design according to Figma

Note: the footer is 100% of the whole screen (and therefore looks a bit too right right) because I couldn't find another way (oh, and I've tried to :))

## screenshots
image

### CODE REVIEW

File: src/locale/en.json [hunk 1]
```diff
@@ -1,4 +1,14 @@
 {
+  "homepage": {
+    "welcome": "Welcome to Databus",
+    "databus_definition": "The open platform for real data on the quality of public transportation lines in Israel",
+    "website_goal": "The purpose of the site is to improve the quality of public transportation in Israel by providing reliable information to journalists, citizens, transportation companies, and government officials in Israel.",
+    "timeline": "Schedule history",
+    "gaps": "Rides that didn't take place",
+    "gaps_patterns": "Patterns",
+    "map": "Time based map",
```
Comment (NoamGaash) [inline src/locale/en.json:9]: we can use same translation keys from the menu, so changing one place will also change the other

**Review by NoamGaash — COMMENTED** [review]
Nice! Thank you :clap: 
But how do I get to this page?
Don't you want to make it the default route?

File: src/locale/en.json (hunk 1 above)
Comment (itsoriki) [reply src/locale/en.json:9]: Done

**Comment by NoamGaash** [comment]
You can add:
```css
.ant-layout {
    position: relative;
}
.container footer {
    position:absolute;
}
main:has(footer) {
    padding-bottom: 40px;
}
```
to fix footer centering

**Review by NoamGaash — APPROVED** [review]

**Pull request merged**